assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -249,18 +128,20 @@ const ChecklistTab = (props) => {
   const renderRow = useCallback(
     (args) => {
       const { id, message, help, pageGroup } = args;
-      const onPrepublish = getOnPrepublishSelect(args);
-      const accessibleText = __('Select offending element', 'web-stories');
+      const prepublishProps = getOnPrepublishSelect(args);
       return (
-        <Row {...onPrepublish} tabIndex={0} key={id} pageGroup={pageGroup}>
-          {onPrepublish.onClick ? (
-            <Underline title={accessibleText}>{message}</Underline>
-          ) : (
-            message
-          )}
-          <HelperText>{help}</HelperText>
-          {onPrepublish.onClick && (
-            <VisuallyHidden>{accessibleText}</VisuallyHidden>
+        <Row key={id} pageGroup={pageGroup}>
+          <IssueTitle>{message}</IssueTitle>
+          <IssueDescription>{help}</IssueDescription>
+          {prepublishProps?.onClick && (
+            <Button
+              variant={BUTTON_VARIANTS.LINK}
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
+              aria-label={TEXT.ACCESSIBLE_LINK_TITLE}
+              {...prepublishProps}
+            >
+              {TEXT.DISPLAY_LINK}
```

Comment:
I don't think the link text is supposed to actually be `"Link"`, but rather some issue-specific CTA like "Edit title" or "Go to cover image" or something like that. The text in Figma is just a placeholder, I assume. But you might want to check with @o-fernandez on that.

Reply:
😂  I'm checking with Sam and Omar about this. I thought `Link` made sense but you're probably right.

Reply:
yup! so it would seem the pre publish panel designs in the design system were never finished or looked at lately by design! whoops! 

chatted offline back and forth with @ivayloklissarov @aaecheverria and @tmagalhaes-huge and arrived at the below agreement for interim designs since prepub updates are on the roadmap for 1.7. This allows the prepub panel to mesh with new designs while Huge works out what this panel will look like/do separately. 

Thanks for flagging, @barklund .

![Screen Shot 2021-03-26 at 11 25 27 AM](https://user-images.githubusercontent.com/10720454/112680900-2fb41100-8e2b-11eb-8966-fad182c3eed0.png)
![Screen Shot 2021-03-26 at 11 25 08 AM](https://user-images.githubusercontent.com/10720454/112680902-304ca780-8e2b-11eb-8588-88b8d22b679b.png)